core/src/test/java/org/springframework/security/authorization/method/AuthorizationManagerBeforeMethodInterceptorTests.java

```diff
@@ -72,7 +72,7 @@ public void beforeWhenMockAuthorizationManagerThenCheck() throws Throwable {
 		AuthorizationManagerBeforeMethodInterceptor advice = new AuthorizationManagerBeforeMethodInterceptor(
 				Pointcut.TRUE, mockAuthorizationManager);
 		advice.invoke(mockMethodInvocation);
-		verify(mockAuthorizationManager).check(any(Supplier.class), eq(mockMethodInvocation));
+		verify(mockAuthorizationManager).authorize(any(Supplier.class), eq(mockMethodInvocation));
```

Comment:
I'd prefer these change to have Mockito call the real `authorize` method, thus confirming that folks upgrading can have the same expectations. For example:

```java
given(mockAuthorizationManager.authorize(any(), any()).willCallRealMethod();
```

Then, this last line in the test shouldn't have to change.

Please see if this strategy works and, if so, apply it to the remaining tests so that their verification steps stay the same.